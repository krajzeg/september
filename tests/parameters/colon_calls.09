identity := |x|{x}
sum := |a,b|{a+b}
printer := |a,b,c|{
	print(a,b,c)
}

fourty_two := identity: 42
print("This should be 42:", fourty_two)

fourty_two = sum: 3*3, 3*11
print("This should be 42 again:", fourty_two)

printer: "And once again", "we expect 42:", 42
