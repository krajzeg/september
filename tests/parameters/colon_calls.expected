This should be 42: 42
This should be 42 again: 42
And once again we expect 42: 42
